package-lock.json

```diff
@@ -32,7 +32,7 @@
         "yauzl": "2.10.0"
       },
       "devDependencies": {
-        "@istanbuljs/nyc-config-typescript": "^1.0.1",
+        "@istanbuljs/nyc-config-typescript": "1.0.1",
```

Comment:
I updated to latest node and ran restore. Want me to keep all of this or revert?

Reply:
Keep it. Better to not let versions drift between dev environments.